component/client-handler/src/main/java/org/wso2/carbon/identity/oauth2/token/handler/clientauth/jwt/validator/JWTValidator.java

```diff
@@ -342,7 +342,7 @@ private OAuthAppDO getOAuthAppDO(String jwtSubject) throws OAuthClientAuthnExcep
         String message = String.format("Error while retrieving OAuth application with provided JWT information with " +
                 "subject '%s' ", jwtSubject);
         try {
-            oAuthAppDO = OAuth2Util.getAppInformationByClientId(jwtSubject);
+            oAuthAppDO = OAuth2Util.getAppInformationByClientIdOnly(jwtSubject);
```

Comment:
Why we need this method change?

Reply:
The currently used method is deprecated. Mentioned it in the PR description as well.

Reply:
This change was reverted in db5a8f354532fa6943464ade0441b0b1d257e36e. Therefore no method change has been done.